resteasy/src/test/java/org/pac4j/jax/rs/rules/RestEasyUndertowServletRule.java

```diff
@@ -66,7 +72,13 @@ protected void before() throws Throwable {
         System.setProperty("org.jboss.resteasy.port", "24257");
         server = new UndertowJaxrsServer().start();
 
-        server.deploy(new MyApp());
+        ResteasyDeployment deployment = new ResteasyDeployment();
+        deployment.setInjectorFactoryClass("org.jboss.resteasy.cdi.CdiInjectorFactory");
+        deployment.setApplication(new MyApp());
+        DeploymentInfo di = server.undertowDeployment(deployment).setContextPath("/").setDeploymentName("DI")
+			.setClassLoader(RestEasyUndertowServletRule.class.getClassLoader())
+			.addListeners(Servlets.listener(org.jboss.weld.environment.servlet.Listener.class));
```

Comment:
do not use fqdn if possible, it is better to have all imports in import section so we can quickly see the dependencies of the class.